clippy_lints/src/methods/mod.rs

```diff
@@ -1693,6 +1706,23 @@ impl<'tcx> LateLintPass<'tcx> for Methods {
             }
         }
     }
+
+    fn enter_lint_attrs(&mut self, cx: &LateContext<'tcx>, attrs: &'tcx [Attribute]) {
```

Comment:
Sure! I guess this can be similar to 

```rust
fn parse_attrs<F: FnMut()>(
    sess: &Session,
    attrs: &[ast::Attribute],
    name: &'static str,
    mut f: F,
) 
```